src/style.css

```diff
@@ -2,0 +1,61 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.body {
+  margin: 0;
+}
```

Comment:
The class name .body is not a good choice. Class names should represent the meaning of the content (not the styles or tag names). In this case, it would be better to use a class name that describes the role of the element in the layout, such as .container, .page, .layout, etc.